tox.ini

```diff
@@ -118,7 +118,7 @@ allowlist_externals =
     bash
 
 setenv =
-    VAULT_ADDR = http://localhost:8200
+    VAULT_ADDR = http://localhost:8201
```

Comment:
not sure if there was a specific port I should've used? I just did +1 from the default port